crates/chia-consensus/src/gen/condition_tools.rs

```diff
@@ -0,0 +1,194 @@
+use crate::consensus_constants::ConsensusConstants;
+use crate::gen::opcodes::{
+    ConditionOpcode, AGG_SIG_AMOUNT, AGG_SIG_ME, AGG_SIG_PARENT, AGG_SIG_PARENT_AMOUNT,
+    AGG_SIG_PARENT_PUZZLE, AGG_SIG_PUZZLE, AGG_SIG_PUZZLE_AMOUNT,
+};
+use crate::gen::owned_conditions::OwnedSpend;
+use chia_protocol::Bytes;
+use chia_protocol::Coin;
+
+pub fn make_aggsig_final_message(
```

Comment:
this is the only public function in this file. Would it not make more sense to call the file `make_aggsig_final_message.rs`?

Reply:
there's also u64_to_bytes in here

Reply:
but it shouldn't be public, should it?

Reply:
It's used in `spendbundle_validation.rs`'s tests and is fairly useful to keep around